test/functional/mongodb/basic/embedded-columns-listeners/entity/Post.ts

```diff
@@ -19,4 +19,7 @@ export class Post {
     @Column(type => Counters)
     counters?: Counters;
 
-}
\ No newline at end of file
+    @Column(type => Counters)
```

Comment:
It's really better to add a new separate test that covers exactly this use case

Reply:
Yeah!